.circle/buildenv_st2.sh

```diff
@@ -52,6 +52,8 @@ fi
 re="\\b$DISTRO\\b"
 [[ "$NOTESTS" =~ $re ]] && TESTING=0
 
-write_env ST2_GITURL ST2_GITREV ST2PKG_VERSION ST2PKG_RELEASE ST2_WAITFORSTART DISTRO TESTING
+ST2_CIRCLE_URL=${CIRCLE_BUILD_URL}
```

Comment:
You don't need this. `CIRCLE_BUILD_URL` is env variable which is already available during build time.

See all Circle env variables: https://circleci.com/docs/environment-variables/